src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/RazorProjectServiceTest.cs

```diff
@@ -1137,13 +1138,80 @@ public async Task AddProject_MigratesMiscellaneousDocumentsToNewOwnerProject()
             x => x.DocumentRemoved(DocumentFilePath2, miscProject.Key));
     }
 
-    private static TextLoader CreateEmptyTextLoader()
+    [Fact]
+    public async Task AddProject_MigratesMiscellaneousDocumentsToNewOwnerProject_FixesTargetPath()
+    {
+        // Arrange
+        const string ProjectFilePath = "C:/path/to/project.csproj";
+        const string IntermediateOutputPath = "C:/path/to/obj";
+        const string DocumentFilePath1 = "C:/path/to/document1.cshtml";
+        const string DocumentFilePath2 = "C:/path/to/document2.cshtml";
+
+        var miscProject = _snapshotResolver.GetMiscellaneousProject();
+
+        await _projectManager.UpdateAsync(updater =>
+        {
+            updater.DocumentAdded(miscProject.Key,
+                new HostDocument(DocumentFilePath1, "C:/path/to/document1.cshtml"), CreateEmptyTextLoader());
+            updater.DocumentAdded(miscProject.Key,
+                new HostDocument(DocumentFilePath2, "C:/path/to/document2.cshtml"), CreateEmptyTextLoader());
+        });
+
+        using var listener = _projectManager.ListenToNotifications();
```

Comment:
Why do you need this listener?

Reply:
I don't, thank you. Left over from copy-and-paste